NetFX build break in Microsoft.IO.Redist

The NetFX and Packaging All Configurations CI builds are failing with the error below. It looks like it might be related to https://github.com/dotnet/corefx/pull/31763. @JeremyKuhne or @MarcoRossignoli, can you please take a look?

> 19:08:23 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\System\IO\DriveInfoInternal.Win32.cs(66,52): error CS0117: 'SR' does not contain a definition for 'Arg_MustBeDriveLetterOrRootDir' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Microsoft.IO.Redist\src\Microsoft.IO.Redist.csproj]
19:08:23 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\System\IO\DriveInfoInternal.Win32.cs(66,84): error CS1503: Argument 2: cannot convert from 'string' to 'System.Exception' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Microsoft.IO.Redist\src\Microsoft.IO.Redist.csproj]
19:08:23 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\System\IO\DriveInfoInternal.Win32.cs(81,48): error CS0117: 'SR' does not contain a definition for 'Arg_MustBeDriveLetterOrRootDir' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Microsoft.IO.Redist\src\Microsoft.IO.Redist.csproj]
19:08:23 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\System\IO\DriveInfoInternal.Win32.cs(81,80): error CS1503: Argument 2: cannot convert from 'string' to 'System.Exception' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Microsoft.IO.Redist\src\Microsoft.IO.Redist.csproj]
19:08:23 D:\j\workspace\windows-TGrou---2a8f9c29\src\System.IO.FileSystem\src\System\IO\File.cs(645,24): error CS0117: 'FileSystem' does not contain a definition for 'Encrypt' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Microsoft.IO.Redist\src\Microsoft.IO.Redist.csproj]
19:08:23 D:\j\workspace\windows-TGrou---2a8f9c29\src\System.IO.FileSystem\src\System\IO\File.cs(650,24): error CS0117: 'FileSystem' does not contain a definition for 'Decrypt' [D:\j\workspace\windows-TGrou---2a8f9c29\src\Microsoft.IO.Redist\src\Microsoft.IO.Redist.csproj]

--- a/src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj
+++ b/src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj
@@ -25,6 +25,7 @@
     <Compile Include="..\..\System.IO.FileSystem\src\System\IO\Error.cs" Link="Microsoft\IO\Error.cs" />
     <Compile Include="..\..\System.IO.FileSystem\src\System\IO\File.cs" Link="Microsoft\IO\File.cs" />
     <Compile Include="..\..\System.IO.FileSystem\src\System\IO\FileInfo.cs" Link="Microsoft\IO\FileInfo.cs" />
+    <Compile Include="..\..\System.IO.FileSystem\src\System\IO\FileSystem.Win32.cs" Link="Microsoft\IO\FileSystem.Win32.cs" />
     <Compile Include="..\..\System.IO.FileSystem\src\System\IO\FileSystem.Windows.cs" Link="Microsoft\IO\FileSystem.Windows.cs" />
     <Compile Include="..\..\System.IO.FileSystem\src\System\IO\FileSystemInfo.cs" Link="Microsoft\IO\FileSystemInfo.cs" />
     <Compile Include="..\..\System.IO.FileSystem\src\System\IO\FileSystemInfo.Windows.cs" Link="Microsoft\IO\FileSystemInfo.Windows.cs" />
@@ -153,6 +154,12 @@
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.GenericOperations.cs">
       <Link>Common\Interop\Windows\Interop.GenericOperations.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.EncryptDecrypt.cs"> 
+      <Link>Common\Interop\Windows\Interop.EncryptDecrypt.cs</Link> 
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.GetVolumeInformation.cs"> 
+      <Link>Common\Interop\Windows\Interop.GetVolumeInformation.cs</Link> 
     </Compile>
     <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\kernel32\Interop.FindFirstFileEx.cs">
       <Link>Common\CoreLib\Interop\Windows\Interop.FindFirstFileEx.cs</Link>

--- a/src/Microsoft.IO.Redist/src/PinvokeAnalyzerExceptionList.analyzerdata
+++ b/src/Microsoft.IO.Redist/src/PinvokeAnalyzerExceptionList.analyzerdata
@@ -1,9 +1,12 @@
+advapi32.dll!EncryptFileW
+advapi32.dll!DecryptFileW
 BCrypt.dll!BCryptGenRandom
 kernel32.dll!CloseHandle
 kernel32.dll!ReplaceFileW
 kernel32.dll!MoveFileExW
 kernel32.dll!DeleteFileW
 kernel32.dll!GetFileAttributesExW
+kernel32.dll!GetVolumeInformationW
 kernel32.dll!CreateDirectoryW
 kernel32.dll!FindNextFileW
 kernel32.dll!DeleteVolumeMountPointW

--- a/src/Microsoft.IO.Redist/src/Resources/Strings.resx
+++ b/src/Microsoft.IO.Redist/src/Resources/Strings.resx
@@ -114,4 +114,7 @@
   <data name="Arg_PathEmpty" xml:space="preserve">
     <value>The path is empty.</value>
   </data>
+  <data name="Arg_MustBeDriveLetterOrRootDir" xml:space="preserve">
+    <value>Drive name must be a root directory (i.e. 'C:\') or a drive letter ('C').</value> 
+  </data>
 </root>

--- a/src/System.IO.FileSystem/src/System/IO/FileSystem.Win32.cs
+++ b/src/System.IO.FileSystem/src/System/IO/FileSystem.Win32.cs
@@ -2,10 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Runtime.InteropServices;
-using System.Text;
 
+#if MS_IO_REDIST
+using System.IO;
+
+namespace Microsoft.IO
+#else
 namespace System.IO
+#endif
 {
     internal static partial class FileSystem
     {

